test/e2e/config/azure-dev.yaml

```diff
@@ -194,7 +194,7 @@ variables:
   KUBERNETES_VERSION_API_UPGRADE_FROM: "v1.26.12"
   FLATCAR_VERSION: "${FLATCAR_VERSION:-latest}"
   ETCD_VERSION_UPGRADE_TO: "3.5.4-0"
-  COREDNS_VERSION_UPGRADE_TO: "v1.9.3"
+  COREDNS_VERSION_UPGRADE_TO: "v1.11.1"
```

Comment:
:+1: matches CAPI on the release-1.6 branch: https://github.com/kubernetes-sigs/cluster-api/blob/90d78fa0c57abacb88a9d158c61a47fe24cda229/test/e2e/config/docker.yaml#L322

Should we maybe bump etcd to `3.5.10-0` too?

Reply:
Good idea.

Ace detective work @nojnhuh!